### DISCUSSION TITLE
Romanian translation for GitVersion docs

### DISCUSSION THREAD

**Discussion by kinkod** [GitTools/GitVersion#3569] [General]
Hi,
Would you be interested in a Romanian translation for GitVersion docs?
One of our SEO clients is willing to sponsor the whole thing in exchange of 2 dofollow backlinks from the docs section of the site.
I'm looking forward to hear from admins.
Thanks

**Comment by arturcic** [comment]
@kinkod question, were the translation is supposed to be hosted? On gitversion.net or somewhere else?

**Reply by kinkod** [reply]
Yes, on gitversion.net

**Comment by asbjornu** [comment]
Which backlinks is it you want to add? Also, do you have a suggestion for how to add translations to the current design and template system? How would you implement it?

**Reply by kinkod** [reply]
Links would be for our clients sites. One is an addictions site and the other a pets site. I haven't checked the source code too well for the docs but seems it is in git. we could replicate the directory and have a /docs/en/ and a /docs/ro/ directory with the romanian translation.
We successfully did a similar translation for semver.org a while ago.

**Reply by kinkod** [reply]
Links would be something like:
Romanian translation made possible by <a>Our Agency Site</a> and sponsored by <a>Client Site 1</a> and <a>Client Site 2</a>.

**Reply by asbjornu** [reply]
@kinkod, could you please be specific about which URLs we would add to the site?

**Reply by kinkod** [reply]
epigrade.com, luxrehabs.com and another one we're not sure about yet.

**Reply by asbjornu** [reply]
>  We successfully did a similar translation for semver.org a while ago.

Were you able to put back-links on SemVer.org as well?

**Reply by kinkod** [reply]
Yes, in credits section at the end.

**Comment by asbjornu** [comment]
@kinkod, thank you so much for the offer to translate the documentation of GitVersion. As GitVersion is a living product and not a static specification (such as SemVer), having to keep and maintain translations of the documentation in several languages is a burden we don't have the resources to take on. Therefore, after careful consideration, we have to respectfully decline.
